helm/values.yaml

```diff
@@ -10,7 +10,7 @@ image:
     tag: latest # The tag value must be passed in via the deploy script
     clientSecret: "" # clientSecret must be passed in via deploy script
   psql:
-    repository: gcr.io/ggl-cas-storage/cas-postgres
+    repository: https://bcgov.github.io/spilo-chart
```

Comment:
this doesn't look like the url for a container image, is it?

Reply:
No, i suppose not. Where would that be found for spilo?

Reply:
isn't spilo the init container?

Reply:
Why are you changing it? Did the init container image for the `psql` client break?

Reply:
Ideally we'd reuse images wherever possible, but we also work in small increments wherever possible. If the psql client didn't break when moving to spilo chart as provider for patroni cluster then let's change as little as possible in this PR

Reply:
There is no obligation to use the same image for the database and the init container. This is merely a convenience (don't need to create a separate dockefile just for `psql`) and optimization (the image will be more likely to be downloaded on the node already, if we use it for the database)